app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class attribute, which means it is shared across all instances of the `Animal` class. This can lead to unexpected behavior when multiple instances are involved. Consider making `alive` an instance attribute or ensure it is accessed correctly using `Animal.alive` if you intend to keep it as a class attribute.